frontend/public/components/_nav.scss

```diff
@@ -33,7 +34,7 @@ $active-border: 3px;
 
 .co-m-nav-link {
   &:hover {
-    background-color: $color-os-nav-background-active;
```

Comment:
Update the value of `$color-os-nav-background-active` to `$color-pf-black-700` so as not to orphan the var and require multiple changes.